drivers/sensor/max30208/Kconfig

```diff
@@ -0,0 +1,103 @@
+# MAX30208 digital temperature sensor
+
+# Copyright (c) 2021, Silvano Cortesi
+# Copyright (c) 2021, ETH Zürich
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig MAX30208
+	bool "MAX30208 Digital Temperature Sensor"
+	depends on I2C
+
+if MAX30208
+
+config MAX30208_FIFO_A_FULL
+	int "FIFO almost full value"
+	range 0 31
+	default 0
+	help
+	  Set the trigger for the FIFO_A_FULL interrupt (e.g. if set to 2,
+	  then the flag is set when the 30th word is written to the FIFO).
+
+config MAX30208_FIFO_ROLLOVER_EN
+	bool "FIFO rolls on full"
+	help
+	  Controls the behavior of the FIFO when the FIFO becomes completely
+	  filled with data. If set, the FIFO address rolls over to zero and the
+	  FIFO continues to fill with new data. If not set, then the FIFO is
+	  not updated until FIFO_DATA is read or the WRITE/READ pointer
+	  positions are changed.
+
+config MAX30208_FIFO_A_FULL_TYPE
+	bool "FIFO almost full type"
+	help
+	  If false, then the A_FULL status bit is asserted on each sample for
+	  which the condition persists. If true, then the bit is asserted only
+	  once until the next A_FULL condition.
+
+config MAX30208_FIFO_STAT_CLR
+	bool "FIFO status clear"
+	help
+	  If true, then A_FULL and TEMP_RDY bit are cleared by a FIFO_DATA register read.
+
+config MAX30208_GPIO0_MODE
+	int "GPIO 0 mode"
+	range 0 3
+	default 2
+	help
+	    Set the mode for GPIO0.
+	    0 = High impedance input
+	    1 = Output (open-drain)
+	    2 = Input with 1MOhm pulldown
+	    3 = Interrupt output (active low)
+
+config MAX30208_GPIO1_MODE
+	int "GPIO 1 mode"
+	range 0 3
+	default 2
+	help
+	    Set the mode for GPIO1.
+	    0 = High impedance input
+	    1 = Output (open-drain)
+	    2 = Input with 1MOhm pulldown
+	    3 = Convert Temperature input (active low)
```

Comment:
these options can likely be moved to devicetree, so that they can be configured per-instance.

Reply:
can they be moved to dev tree but still have a Kconfig being dependend on them (interrupt requires GPIO0 mode being 3)?